datasets/usa_names/usa_1910_current/pipeline.yaml

```diff
@@ -22,6 +22,7 @@ resources:
       source: http://www.ssa.gov/OACT/babynames/limits.html
 
 dag:
+  airflow_version: 2
```

Comment:
Just double checking this one is a 2

Reply:
Yup, that's the first pipeline that's migrated over to use Airflow 2 operators.

Reference: #136